### PR TITLE
[Kaptain] Fixed links & fine-tuning

### PR DESCRIPTION
**Pull request by gvasquezvargas**
## Jira Ticket
Fixing last details 

### Preview
http://docs-d2iq-com-pr-4368.s3-website-us-west-2.amazonaws.com/

### CODE REVIEW

**Review by juliangieseke — APPROVED** [review]

**Review by timcharper — APPROVED** [review]

**Review by mmcelroy91 — APPROVED** [review]

File: pages/dkp/kaptain/2.0.0/install/deploy-kaptain/index.md [hunk 1]
```diff
@@ -12,6 +12,14 @@ enterprise: false
 You can deploy Kaptain to a cluster in a selected workspace. If you do not intend to deploy Kaptain to a certain cluster, you must switch the workspace you are deploying to or move that cluster to another workspace.
 </p>
 
+## Prerequisite
+
+Ensure you have added Kaptain to your DKP Catalog applications before you deploy it to your clusters. Refer to the corresponding documentation depending on your environment:
```
Comment (Colin1841) [inline pages/dkp/kaptain/2.0.0/install/deploy-kaptain/index.md:17]:
```suggestion
Ensure you add Kaptain to your DKP Catalog applications before you deploy it to your clusters. Refer to the corresponding documentation for your environment:
```

File: pages/dkp/kaptain/2.0.0/install/dkp/index.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ enterprise: false
 ---
 
 <p class="message--warning"><strong>WARNING: </strong>
-You can deploy Kaptain to a cluster in a selected Workspace. If you do not intend to deploy Kaptain to a certain cluster, you must switch the Workspace you are deploying to or move that cluster to another Workspace.
+You can deploy Kaptain to a cluster in a selected workspace. If you do not intend to deploy Kaptain to a certain cluster, you must switch the workspace you are deploying to or move that cluster to another workspace.
 </p>
```
Comment (Colin1841) [inline pages/dkp/kaptain/2.0.0/install/dkp/index.md:13]: This is one of those things we'll need to clean up eventually - we're capitalizing Workspace in some locations. We'll want the product team to weigh in on this, I suppose.

File: pages/dkp/kaptain/2.0.0/install/dkp/index.md [hunk 3]
```diff
@@ -40,18 +40,18 @@ The amounts depend on the number, complexity, and size of the workloads, in addi
...
   - Istio
-  - Knative
+  - Knative (optional, if KServe is configured to work in `RawDeployment` mode)
 
 - [`kubectl`][kubectl] on your installation machine
 
 - For customers deploying in a multi-cluster environment (Enterprise): Ensure you have configured [Kaptain to authenticate with a Management Cluster][dex].
 
-- For DKP 2.x, ensure the following applications are enabled in Kommander:
+- Ensure the following applications are enabled in Kommander:
```
Comment (Colin1841) [inline pages/dkp/kaptain/2.0.0/install/dkp/index.md:54]:
```suggestion
- Ensure you enable the following applications in Kommander:
```

**Review by Colin1841 — APPROVED** [review]

File: pages/dkp/kaptain/2.0.0/install/deploy-kaptain/index.md [hunk 4]
```diff
@@ -20,7 +30,7 @@ Follow these steps to enable Kaptain in air-gapped and networked environments fr
 
 1.  Select **Applications** from the sidebar menu.
 
-1.  Type Kaptain in the **Applications** search bar. If Kaptain is not available in the UI, [add Kaptain to your DKP catalog][add_Kaptain] via CLI first.
+1.  Type Kaptain in the **Applications** search bar. If Kaptain is not available in the UI, add Kaptain to your DKP catalog as stated in the [prerequisites][#prerequisite].
```
Comment (gvasquezvargas) [inline pages/dkp/kaptain/2.0.0/install/deploy-kaptain/index.md:33]:
```suggestion
1.  Type Kaptain in the **Applications** search bar. If Kaptain is not available in the UI, add Kaptain to your DKP catalog as stated in the [prerequisite section](#prerequisite).
```

**Pull request merged**